tests/pw/pages/productAddonsPage.ts

```
@@ -81,13 +92,44 @@
+        await this.toBeVisible(addonsVendor.addon.addonRow(addonTitle));
+        await this.saveAddon();
+        await this.toBeVisible(addonsVendor.addon.addonRow(addonTitle));
+    }
+
+    // export addon field
+    async exportAddonField(addonId: string, addon: any): Promise<void> {
+        await this.goToAddonEdit(addonId);
+        await this.click(addonsVendor.addon.export);
+        const exportedAddon = unserialize((await this.getElementText(addonsVendor.addon.exportInput))!);
+        exportedAddon[0].options.forEach((option: { min: any; max: any }) => {
+            delete option.min;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)